boa/src/builtins/object/mod.rs

```diff
@@ -78,6 +78,8 @@ pub enum ObjectData {
     Symbol(RcSymbol),
     Error,
     Ordinary,
+    Date(RcDate),
```

Comment:
I don't need `RcData` at all, since it just stores a `Rc<Data>` and the thing that `Data` stores is copyable, so why not just store `ObjectData::Data(Data)` and copy it if we need it instead of keeping a reference counted pointer to an allocation.

Reply:
Good point! I should watch what I copy :).